server/src/mongodb/index.js

```diff
@@ -7,6 +7,9 @@ let _client;
 module.exports = {
   connect: () => {
     MongoClient.connect(url, function(err, client) {
+      if(err){
+        return console.log(err);
```

Comment:
The nodejs mongo driver will automatically retry the 'connect' function 30 times (once a second).  Adding a catch for 'err' and returning before calling 'db' on an empty client seems to have resolved all timing issues I was having when starting up the project.   Returning console.log might be a bit of a hack, but since this is just a function definition used by the internals of 'connect' I personally didn't see a problem.  Let me know your thoughts, I can break it out if you'd like.